spring-cloud-gcp-docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -80,15 +77,19 @@ The Spring Boot starter for Google Cloud Pub/Sub auto-configures a `SubscriberFa
 
 ==== Pulling messages from a subscription
 
-Google Cloud Pub/Sub supports the synchronous pulling of messages from a subscription.
+Google Cloud Pub/Sub supports synchronous pulling of messages from a subscription.
 This is different from subscribing to a subscription, in the sense that subscribing is an
 asynchronous task which polls the subscription on a set interval.
 
-The `pullNext()` method allows for a single message to be pulled from a subscription.
-The `pull()` method pulls a number of messages from a subscription, allowing for the retry settings
-to be configured.
+The `pullNext()` method allows for a single message to be pulled and automatically acknowledged from a subscription.
+The `pull()` method pulls a number of messages from a subscription, allowing for the retry settings to be configured.
```

Comment:
I think it's worth saying here that `pull()` does not auto-ack, but returns a list of `AcknowledgeablePubsubMessage`s